src/backend/api/reservations.js

```diff
@@ -0,0 +1,58 @@
+const express = require("express");
+const router = express.Router();
+const knex = require("../database");
+
+router.get("/", async (request, response) => {
+    try {
+        const reservations = await knex("reservation");
+        response.json(reservations);
+    } catch (error) {
+        throw error;
+    }
+});
+
+router.post("/", async (request, response) => {
+    try {
+        await knex("reservations").insert(request.body);
+        response.json("Reservation is added");
+    } catch (error) {
+        throw error
+    }
+
+});
+
+router.get("/:id", async (request, response) => {
+    try {
+        const reservationsById = await knex('reservation')
+            .where({ id: parseInt(request.params.id) });
+        response.json(reservationsById);
+    } catch (error) {
+        throw error;
+    }
+});
+
+
+router.put('/:id', async (request, response) => {
+    try {
+        const updatedReservations = await knex('reservation')
+            .where({ id: parseInt(request.params.id) })
+            .update(request.body);
+        response.json('Reservation updated');
+    } catch (error) {
+        throw error;
+    }
+});
+
+
+router.delete('/:id', async (request, response) => {
+    try {
+        const deletedReservation = await knex('reservation')
+            .where({ id: parseInt(request.params.id) })
+            .del();
+    } catch (error) {
+        throw error;
+    }
+    response.json('Reservation deleted');
+});
+
+module.exports = router;
```

Comment:
This and the other files should end with a newline. That's why GitHub shows this symbol:

<img width="40" alt="Screen Shot 2021-05-24 at 12 10 15" src="https://user-images.githubusercontent.com/809707/119332539-0546ce80-bc89-11eb-97e7-7eb6bad8c447.png">